cli/jobs/pipelines/automl/cli-automl-classification-task-bankmarketing-pipeline/components/component_preprocessing.yaml

```diff
@@ -0,0 +1,26 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+name: data_preprocessing
+display_name: Data Preprocessing
+version: 1
+type: command
+inputs:
+  train_data:
+    type: mltable
+  validation_data:
+    type: mltable
+outputs:
+  preprocess_train_data:
```

Comment:
>preprocess [](http://example.com/codeflow?start=2&length=10)

`preprocessed_train_data` & `preprocessed_validation_data`?

Reply:
Done. Thanks for pointing this out :)